HISTORY.md

```diff
@@ -8,6 +8,7 @@
 * Fix a bug when format_version=3, partitioned fitlers, and prefix search are used in conjunction. The bug could result into Seek::(prefix) returning NotFound for an existing prefix.
 * Revert the feature "Merging iterator to avoid child iterator reseek for some cases (#5286)" since it might cause strong results when reseek happens with a different iterator upper bound.
 * Fix a bug causing a crash during ingest external file when background compaction cause severe error (file not found).
+* Fix a bug when partitioned fitlers and prefix search are used in conjunction, ::SeekForPrev could return invalid for an existing prefix.
```

Comment:
This description is misleading. Users don't need to explicitly call SeekForPrev() to see the bug. It can be called inside RocksDB for other operations. For example, the failure we saw was a Seek() followed by Prev().

Reply:
I updated that to include Seek+Prev, and Seek that involves Delete or Merge operands.